templates/partials/_video.html

```diff
@@ -0,0 +1,11 @@
+{% if video.type == "youtube" %}
+  <iframe id="ytplayer" type="text/html" width="643" height="362"
+          src="{{ video.url }}?autoplay=1&mute=1&loop=1&modestbranding=1&playlist={{ video.id }}" frameborder="0"></iframe>
+{% elif video.type == "vimeo" %}
+  <iframe id="vimeoplayer" width="643" height="362" frameborder="0"
+          webkitallowfullscreen mozallowfullscreen allowfullscreen
+          src="{{ video.url }}?loop=1&title=0&byline=0&portrait=0"></iframe>
```

Comment:
Should it have `autoplay=1`? https://help.vimeo.com/hc/en-us/articles/115004485728-Autoplaying-and-looping-embedded-videos

Or maybe it's my chrome blocking autoplay...

Reply:
Should it be `muted=1`?

Reply:
Yes, it's Chrome blocking autoplay - works fine in Firefox.

Reply:
That article is interesting, it didn't work when I tried adding ?muted=1 and found some documentation saying it wasn't supported.... I'll try again.